client/src/utils/timerUtil.test.js

```diff
@@ -0,0 +1,16 @@
+import { convertSecToMinSecStr } from './timeUtil';
+
+describe('convertSecToMinSecStr: Test timer convertion', () => {
+  it('Should handle valid time', () => {
+    expect(convertSecToMinSecStr(140).toBe('02:20'));
+    expect(convertSecToMinSecStr(719)).toBe('11:59');
+    expect(convertSecToMinSecStr(0)).toBe('00:00');
+  });
+
+  // TODO: test negative cases after test framework is integrated.
```

Comment:
Please handle all the cases here in this PR only, as todos will get lined up very easily. Try to avoid that.